[Components - FontSizePicker]: Use incremental sequence of numbers as labels for the available font-sizes at the segmented control (conditionally) (#37038)
WordPress/gutenberg: packages/components/src/font-size-picker/README.md

### fontSizes

> An array of font size objects. The object should contain properties `size`, `name`, `slug` and can contain `alias` on special use cases described right below.

Reviewer: Feel free to ignore this suggestion. It's my attempt to make this read a little better, so it's completely subjective.

```suggestion
An array of font size objects. Each object should contain the properties `size`, `name`, and `slug`. Optionally, `alias` can be included for special use cases as described below.
```
